classes/local/object_manipulator/deleter.php

```diff
@@ -81,13 +81,13 @@ protected function get_candidates_sql() {
                        f.contenthash,
                        MAX(f.filesize) AS filesize
                   FROM {files} f
-             LEFT JOIN {tool_objectfs_objects} o ON f.contenthash = o.contenthash
+            INNER JOIN {tool_objectfs_objects} o ON f.contenthash = o.contenthash
                  WHERE o.timeduplicated <= ?
                        AND o.location = ?
+                       AND f.filesize > ?
```

Comment:
more whitespace, and the unchanged line above it

Reply:
re the whitespace indent, this is slightly confusing as the wiki page example is kinda ambiguous or misleading.

https://docs.moodle.org/dev/SQL_coding_style#Indentation

In my book the AND should be left of the red line, like this:

https://github.com/moodle/moodle/blob/master/admin/tool/cohortroles/classes/api.php#L210

Core is all over the shop so it's easy to find examples both ways. HQ need a better policy and to update the wiki

Reply:
fyi https://tracker.moodle.org/browse/MDL-49238?focusedCommentId=683604&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-683604